sihl_core/lib/db.ml

```diff
@@ -249,19 +286,22 @@ dirty = %bool{dirty}
       match steps with
       | [] -> Lwt_result.return ()
       | (name, query) :: steps -> (
-          Lwt_io.printf "Running: %s\n" name >>= fun () ->
+          Logs_lwt.info (fun m -> m "Running: %s\n" name) >>= fun () ->
```

Comment:
@aronerben Because I have inconsistently bound `let*` to `Lwt_result.bind` and `Lwt.bind`. I will fixed that and use `let (let*) = Lwt.bind` everywhere.